src/viking.js

```diff
@@ -1,11 +1,59 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+constructor(health, strength) {
+    this.health = health; 
+    this.strength = strength; 
+}
+attack(){
+    return this.strength; 
+}
+receiveDamage (theDamage){
+    this.health -= theDamage; 
+}
+}
 // Viking
-class Viking {}
+
+class Viking extends Soldier {
+    constructor (name, health, strength){
+        super(health, strength); 
+        this.name = name
+    }
+    receiveDamage (theDamage){
+        this.health -= theDamage; 
+        if (this.health>0) {
+            return this.name + 'has received' + theDamage + 'point of damage';
```

Comment:
para satisfacer el test, solo te hace falta añadir un espacio entre los valores y las frase ;) 
`this.name + ' has received' + theDamage + ' points of damage';`